ui/src/app/pages/admin/user/admin-user-table.tsx

```diff
@@ -280,7 +280,8 @@ export const AdminUserTable = withUserProfile()(
                 frozenWidth='200px'
                 globalFilter={filter}
                 paginator={true}
-                rows={50}
+                rows={5}
+                rowsPerPageOptions={[5, 10, 50, 100]}
```

Comment:
Why 5 rows? Because I see half of page is blank.

Reply:
10 records shows up on my monitor without scrolling but 10 (and even a little with 5) causes scrolling on my laptop screen. I changed the default to 10.